conf/messages.en

```diff
@@ -249,4 +249,10 @@ ultimateParentCompanyInternationalAddress.checkYourAnswersLabel = UltimateParent
 ultimateParentCompanyUkAddress.title = What is the registered office address of {0}?
 ultimateParentCompanyUkAddress.heading = What is the registered office address of {0}?
 ultimateParentCompanyUkAddress.buildingOrStreet.change.hidden = buildingOrStreet
-ultimateParentCompanyUkAddress.postcode.change.hidden = postcode
\ No newline at end of file
+ultimateParentCompanyUkAddress.postcode.change.hidden = postcode
+checkUltimateGlobalParentCompanyInUk.title = Ultimate parent company details
+checkUltimateGlobalParentCompanyInUk.heading = is the registered office address of {0} in the UK?
+checkUltimateGlobalParentCompanyInUk.checkYourAnswersLabel = checkUltimateGlobalParentCompanyInUk
+checkUltimateGlobalParentCompanyInUk.error.required = Select yes if checkUltimateGlobalParentCompanyInUk
```

Comment:
Not sure if this is the right place. When nothing is selected and you click `Continue`, the error message appears as
`Select yes if checkUltimateGlobalParentCompanyInUk`